--- vif     2010-12-20 16:39:46.000000000 +0000
+++ vif_modified 2010-11-19 23:24:37.000000000 +0000
@@ -213,6 +213,7 @@
 
         # xs-xen.pq.hq:91e986b8e49f netback-wait-for-hotplug
         xenstore-write "/local/domain/0/backend/vif/${DOMID}/${DEVID}/hotplug-status" "connected"
+       python /etc/xensource/scripts/vif_rules.py ${DOMID} online 2>&1 > /dev/null
     fi
     ;;
 
@@ -224,6 +225,7 @@
 
 remove)
     if [ "${TYPE}" = "vif" ] ;then
+       python /etc/xensource/scripts/vif_rules.py ${DOMID} offline 2>&1 > /dev/null
         xenstore-rm "${HOTPLUG}/hotplug"
     fi
     logger -t scripts-vif "${dev} has been removed"
